src/Styles/main.css

```diff
@@ -9,8 +9,6 @@
 
 /* Navigation */
 .navBar {
-  background-color: var(--darc);
-  color: var(--snow);
   display: flex;
   justify-content: space-between;
   align-items: center;
```

Comment:
You can use a utility class for this so that it can be used at multiple places.

Reply:
right, i have created a utility class, i will refactor the code in the next pr, thanks